website/templates/project/project_base.mako

```diff
@@ -1,5 +1,25 @@
 <%inherit file="../base.mako"/>
 
+<%def name="resource()"><%
+        if node.get('is_registration', False):
+           return 'registration'
+        elif node.get('is_preprint', False):
+           return 'preprint'
+        elif parent_exists:
+            return 'component'
+        else:
+            return 'project'
+    %>
+</%def>
+
+<%def name="public()"><%
+        if node.get('is_public', False):
```

Comment:
Haven't looked into why but public registrations still send is_public=False to GA.

Reply:
If your registration is still pending it remains private. Typically this happens until all contributors approve the registration or 48 hours elapse. I think this probably what's happening in your case, it caught me off guard as well.

Reply:
ahhh, right.

Reply:
Minor: this can be simplified to `return node.get('is_public', False)`.

I can do this when I merge.